gap/profiling.gi

```diff
@@ -783,13 +792,13 @@ function(data, outfile, jobid, pathtoremove)
     prev := false;
 
     for file in data.line_info do
-        if file[1] <> "stream" then
+        if IsExistingFile(file[1]) then
```

Comment:
Could the same fix please be applied to the codecov generator function a few lines above this one?

Reply:
Sure, though strictly speaking this is not necessary for codecov output (as we don't care whether the file currently exists).

I wonder whether we can compute the `md5sum` for a file and put that into the profiling output?